reference/other.rst

```diff
@@ -0,0 +1,14 @@
+.. _conan_ref_other:
+
+Other
+=====
+
+These are other features that Conan also implements:
+
+
+Contents:
+
+.. toctree::
+   :maxdepth: 1
+
+   other/backup_sources
```

Comment:
I don't love the ``other`` section, I think we need to look for a better naming (I have tried a bit without success...)

Reply:
This feature should definitely have an entry in https://docs.conan.io/2/tutorial/other_features.html, but we still need to find a place for it.